eng/pipelines/official.yml

```diff
@@ -1,191 +1,115 @@
+  type: object
+  default:
+  - rel/d17.13
 
-# Note: Only add pipeline variables if they apply to most of the stages/jobs.
 variables:
+# Variables used: DncEngInternalBuildPool
 - template: /eng/common/templates-official/variables/pool-providers.yml@self
-  # The configuration for the build is used throughout the various pipelines as the file paths for output contain the build configuration as a folder name.
-- name: BuildConfiguration
-  value: Release
-  # https://docs.microsoft.com/dotnet/core/tools/dotnet-environment-variables#dotnet_skip_first_time_experience
-- name: DOTNET_SKIP_FIRST_TIME_EXPERIENCE
-  value: true
-  # https://docs.microsoft.com/dotnet/core/tools/dotnet-environment-variables#dotnet_nologo
-- name: DOTNET_NOLOGO
-  value: true
-  # https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/26284/Enabling-SBOM-For-Your-Component-Insertion-into-VS?anchor=1.-add-the-%27manifest-generator-task%27-to-your-pipeline
-- name: packaging.EnableSBOMSigning
-  value: true
-  # Opt out of automatically injecting Codesign Validation into the pipeline. We run Codesign Validation as part of the Compliance pipeline.
-  # See: https://aka.ms/gdn-injection
-- name: runCodesignValidationInjection
-  value: false
-  # Suspend enforcement of NuGet Single Feed Policy. See:
-  # - https://aka.ms/nugetmultifeed
-  # - https://docs.opensource.microsoft.com/tools/nuget_security_analysis/nuget_security_analysis/
-  # - https://docs.opensource.microsoft.com/tools/cg/how-to/nuget-multifeed-configuration/
-  # - https://onebranch.visualstudio.com/OneBranch/_wiki/wikis/OneBranch.wiki/5205/TSG-Build-Broken-Due-to-Using-Multiple-Feeds?anchor=setting-nugetsecurityanalysiswarninglevel-in-cdp
-- name: NugetSecurityAnalysisWarningLevel
-  value: none
-  # Allows CodeQL to run on our Build job.
-  # https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/configuring-codeql3000-ado-pipelines
-- name: Codeql.Enabled
-  value: true
-  # Default to skipping auto-injection for CodeQL. It is not skipped in the Build job only.
-  # https://eng.ms/docs/cloud-ai-platform/devdiv/one-engineering-system-1es/1es-docs/codeql/configuring-codeql3000-ado-pipelines#monolithic-repos-and-multistage-pipelines
-- name: Codeql.SkipTaskAutoInjection
-  value: true
-- ${{ if eq(variables['_RunAsPublic'], 'true') }}:
-  - name: _AdditionalBuildArgs
-    value: ''
-  - name: _BuildJobDisplayName
-    value: 'Build and Test'
-- ${{ else }}:
-  - name: _AdditionalBuildArgs
-    value: '/p:Test=false'
-  - name: _BuildJobDisplayName
-    value: 'Build, Sign and Publish'
+############### ARCADE ###############
+# Both this (used in Arcade for the MicroBuildSigningPlugin) and DotNetSignType (used in Arcade in Sign.proj) are necessary to set the sign type.
+# https://github.com/dotnet/arcade/blob/ccae251ef033746eb0213329953f5e3c1687693b/Documentation/ArcadeSdk.md#common-steps-in-azure-devops-pipeline
+- name: _SignType
```

Comment:
Is this only listed under secondaryVsInsertionBranches? Does that mean the primary one isn't signed?

Reply:
I don't know what you mean by "listed under". It is the pipeline variable used in the build to set the sign type for the build. You can see the usage when we call `build.ps1` inside of `workload-build.yml`.

Reply:
If I understand it correctly, the indentation here means that this applies to secondaryVsInsertionBranches but not primary...if it applies to the whole pipeline (i.e., both), then I was just wrong

Reply:
`-` means it is an element in an array. The array is defined by the `variables:` label. So, it is an element in `variables`. `-` can either exist at the same indentation level of the label (aligned with the start of `variables:`) or as a 2-space indentation. Either will work for it (it is a stylistic choice).